Topic 0/src/Classes/Builder.java

```diff
@@ -0,0 +1,56 @@
+package Classes;
+
+public abstract class Builder {
+	public class Home {
+
+	}
+
+	private int rooms;
+	private int windows;
+	private String street;
+	private String phone;
+	
+	public Builder(int r, int w, String s, String p) {
+		setRooms(r);
+		setWindows(w);
+		setStreet(s);
+		setPhone(p);
+	}
+	
+	public abstract void printout();
```

Comment:
Why this is an abstract method? You can define it just printing all attributes for this Builder.